### PR TITLE
Simplified build for proxy

### PR DESCRIPTION
**Pull request by rhim**
1. Removed extraneous UI build steps.
2. Removal of unnecessary files.
3. A simple local deployment to test out proxy only functionality.

Signed-off-by: Himanshu Raj <rhim@users.noreply.github.com>

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -49,52 +49,6 @@ else
 fi
 
 #
-# run `npm install` to prepare the UI if it's changed since the last time we ran it
-#
-newest_commit=$(git rev-parse HEAD)
-last_commit_file="../.last_npm_installed_ui_commit"
```
Comment (yuva29) [inline scripts/build.sh]: Are we sure we don't need this anymore? This file is referred here https://github.com/contiv/auth_proxy/blob/master/scripts/build.sh#L55 

I guess without `last_npm_installed_ui_commit ` npm will be installed every time we build.

File: build/Dockerfile.base [hunk 2]
```diff
@@ -1,18 +0,0 @@
-FROM ubuntu:16.04
```
Comment (yuva29) [inline build/Dockerfile.base]: Also, this image is used for running tests and go checks. https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/checks.sh#L7 
https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/systemtests.sh#L34
https://github.com/contiv/auth_proxy/blob/28b32e91762d363ac41569efc8fe7988a4488ece/scripts/unittests.sh#L26

If we're removing this, it's needs to propagated everywhere.

File: scripts/build.sh (hunk 1 above)
Comment (rhim) [inline scripts/build.sh]: I am not sure I understand - that reference is what is being deleted. Am I missing something?

File: build/Dockerfile.base (hunk 2 above)
Comment (rhim) [inline build/Dockerfile.base]: Good catch! Let me go fix these as well.

File: scripts/build.sh (hunk 1 above)
Comment (rhim) [inline scripts/build.sh]: and we don't need npm install anymore. Latest UI changes remove that requirement.

File: build/Dockerfile.base (hunk 2 above)
Comment (rhim) [inline build/Dockerfile.base]: Cleaned up. PTAL.

**Pull request merged**